.gitignore

```diff
@@ -5,20 +5,5 @@ tests/cov/
 tests/temp
 tests/.phpunit.result.cache
 
-#vim
-.*.swp
-
-#binaries
-bin/phpunit
-bin/php-cs-fixer
-bin/phpstan
-bin/phpstan.phar
-bin/hoa
-
 # Development stuff
-testdata/
 .php_cs.cache
-.idea
```

Comment:
This is handy to have.
When using PHPstorm IDE, it creates a folder called `.idea` 
Without this in `.gitignore` it is easy to accidentally commit the folder.

Reply:
I think that files and folders created by the developer's environment should be in the [global .gitignore](https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) of that developer. Otherwise, one would need to add so much more files and folders to support the many editors and other tools used.

Reply:
How have I gone this long without anyone telling me this? And seen so many repos that have patterns like gthis listed in their `.gitignore`?

Thanks for educating me.